app/uk/gov/hmrc/trackingconsentfrontend/views/CookieSettingsPage.scala.html

```diff
@@ -19,12 +19,12 @@
         settingsPageScripts: SettingsPageScripts,
         cookieSettingsIntro: CookieSettingsIntro,
         unsupportedBrowser: UnsupportedBrowser,
-        supportedBrowser: SupportedBrowser)
-
+        supportedBrowser: SupportedBrowser,
+        optimizelySnippet: OptimizelySnippet)
 @()(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
 
 @defining(messages("cookieSettings.title")) { title =>
-    @layout(pageTitle = Some(title), scriptsBlock = Some(settingsPageScripts())) {
+    @layout(pageTitle = Some(title), headBlock = Some(optimizelySnippet()), scriptsBlock = Some(settingsPageScripts())) {
```

Comment:
We'll need to change this so the settingsPageScripts get loaded in the HEAD before the optimizely snippet, perhaps introduce a new template that includes settingsPageScripts first and then optimizelySnippet. scriptsBlock adds scripts to the bottom of the body tag.

Reply:
Updated, let me know what you think of the new version.